test-engineering/contract-tests/volumes/client/scenarios.yml

```diff
@@ -14,6 +14,7 @@ scenarios:
         response:
           status_code: 200
           content:
+            sov: 'eyJuYW1lIjoiU09WLTIwMjMwNTE4MjE1MzE2IiwiYWxsb2NhdGlvbnMiOlt7InBvc2l0aW9uIjoxLCJhbGxvY2F0aW9uIjpbeyJwYXJ0bmVyIjoiYW1wIiwicGVyY2VudGFnZSI6MTAwfV19LHsicG9zaXRpb24iOjIsImFsbG9jYXRpb24iOlt7InBhcnRuZXIiOiJhbXAiLCJwZXJjZW50YWdlIjo4OH0seyJwYXJ0bmVyIjoibW96IiwicGVyY2VudGFnZSI6MTJ9XX1dfQ'
```

Comment:
You can break long line into multilines with `>` in yaml.
```
sov: >
  foo
  bar
  ...
```

Reply:
Oh you can also create an anchor for this string literal and reference it elsewhere.

```yaml
sov: &sov >
  foo
  bar
  ...

# for other records
sov: *sov
```